README.md

```diff
@@ -1 +1,346 @@
-#  Chart repo
+# OpenShift Helm Charts Repository
+
+OpenShift Helm Charts Repository is the canonical source of all the certified
+Helm charts from [partners][partners] and the community. Once your chart is
+ready for submission, you can submit it as a pull request. The chart is going to
+be certified after running the [chart-verifier][chart-verifier] tool against
+it. You may also consider running the chart-verifier locally and submit the
+generated report for certification.
+
+After merging your pull request, the CI/CD pipeline will publish the chart in
+the GitHub releases and add it to the Helm [charts repository
+index][index-url]. If you are submitting the report without a chart, the report
+must contain a URL pointing to the downloadable location of the chart. In this
+case, the Helm charts repository index will include the download URL and other
+chart metadata available in the report. If you want the chart published through
+GitHub but do not want the CI/CD pipeline running chart-verifier against your
+chart, you can submit a chart and the report together.
+
+---
+
+<!-- To generate: gh-md-toc --no-backup --hide-footer README.md -->
+
+**Table of Contents:**
+
+<!--ts-->
+* [OpenShift Helm Charts Repository](#openshift-helm-charts-repository)
+   * [Submitting a Chart](#submitting-a-chart)
+      * [Preparation](#preparation)
+      * [Chart Source](#chart-source)
+      * [Chart Verification Report](#chart-verification-report)
+      * [Chart with Verification Report](#chart-with-verification-report)
+   * [Troubleshooting Pull Request Failures](#troubleshooting-pull-request-failures)
+      * [Error when submitting files not part of any chart](#error-when-submitting-files-not-part-of-any-chart)
+      * [Pull request author is not part of OWNERS file](#pull-request-author-is-not-part-of-owners-file)
+      * [Vendor label mismatch with the directory structure](#vendor-label-mismatch-with-the-directory-structure)
+      * [Chart name mismatch with the directory structure](#chart-name-mismatch-with-the-directory-structure)
+      * [Error when submitting both chart source and tarball](#error-when-submitting-both-chart-source-and-tarball)
+      * [Error when submitting files not related to any chart](#error-when-submitting-files-not-related-to-any-chart)
+      * [Chart name and version mismatch errors](#chart-name-and-version-mismatch-errors)
+      * [Report failures](#report-failures)
+   * [Frequently Asked Questions](#frequently-asked-questions)
+      * [Can I test the pull request in my fork before submitting?](#can-i-test-the-pull-request-in-my-fork-before-submitting)
+      * [Can I use any command-line interface to create pull request?](#can-i-use-any-command-line-interface-to-create-pull-request)
+      * [How to update OWNERS file?](#how-to-update-owners-file)
+   * [Support](#support)
+<!--te-->
+
+---
+
+## Submitting a Chart
```

Comment:
Would like to see here a summary of the submission options available:
- Just a report
- Chart tarball with or without report
- Chart src with or without report.

They need to be given equal weight.